src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -293,11 +293,10 @@ public void Track(ITelemetry telemetry)
             if (this.IsEnabled())
             {
                 this.Initialize(telemetry);
-                
-                this.WriteTelemetryToDebugOutput(telemetry);
```

Comment:
I am thinking of output window is a diagnostics tool for AI. I would like to see what telemetry items were filtered out (by any processor). So it would be super helpful if I can see all filtered telemetry items as
[Filtered out - not sent to server]: {telemetry item}